app/forms/trip_form.py

```diff
@@ -0,0 +1,9 @@
+from flask.ext.wtf import Form
+from wtforms import StringField, SubmitField
+from wtforms.validators import ValidationError, DataRequired
+from app.models import User
+
+class TripForm(Form):
+    name = StringField('Trip Name', render_kw={'class': 'form-control', 'placeholder': 'Spring Break Gppd Luck with Your Liver', 'required': True}, validators=[DataRequired('Please enter your trip name.')])
```

Comment:
The names of the fields in the `TripForm` should correspond to those in the `Trip` model you can find in **models/Trip.y**. In this particular case change `name` to `title`.

- [ ] Should be:
```
title = StringField('Trip Name', render_kw={'class': 'form-control', 'placeholder': 'Spring Break Gppd Luck with Your Liver', 'required': True}, validators=[DataRequired('Please enter your trip name.')])
```

**Note:** This will also require you to change the corresponding references to the _name_ property to _title_ in the 
`create_trip()` method in **views.py** and the *templates/createTrip.html* file.